src/NoteDisplay.cpp

```diff
@@ -797,7 +797,21 @@ void NoteDisplay::DrawHoldPart(vector<Sprite*> &vpSpr,
 	// shifted by one pixel or there is a seam. -Kyz
 	if(part_type == hpt_bottom)
 	{
-		add_to_tex_coord= SCALE(1.0f, 0.0f, power_of_two(unzoomed_frame_height), 0.0f, 1.0f);
+		if (!part_args.anchor_to_top)
+		{
+			float offset = unzoomed_frame_height - (y_end_pos - y_start_pos);
+			// �����O�m�[�g�{�̂̒�����unzoomed_frame_height��0�̂Ƃ��ɁAadd_to_tex_coord��0��1�ɂ����OK
+			// �܂�Aoffset��0��unzoomed_frame_height�ɂ���Ɨ��z�ʂ�̕\���ɂȂ� -A.C
```

Comment:
Shift texture coord to fit hold length If hold length is less than bottomcap frame size